src/status_im2/contexts/quo_preview/avatars/account_avatar.cljs

```diff
@@ -1,37 +1,52 @@
 (ns status-im2.contexts.quo-preview.avatars.account-avatar
-  (:require [quo2.components.avatars.account-avatar :as quo2]
+  (:require [quo2.components.colors.color-picker.view :as color-picker]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [reagent.core :as reagent]
             [status-im2.contexts.quo-preview.preview :as preview]))
 
 (def descriptor
-  [{:label   "Icon"
-    :key     :icon
+  [{:label   "Type"
+    :key     :type
     :type    :select
-    :options [{:key   :main-icons/wallet
-               :value "Wallet"}
-              {:key   :main-icons/token
-               :value "Token"}
-              {:key   :main-icons/status
-               :value "Status"}]}
+    :options [{:key   :default
+               :value "default"}
+              {:key   :watch-only
+               :value "watch only"}]}
    {:label   "Size"
     :key     :size
     :type    :select
-    :options [{:key   20
-               :value "Small"}
+    :options [{:key   16
+               :value "16"}
+              {:key   20
+               :value "20"}
               {:key   24
-               :value "Medium"}
+               :value "24"}
+              {:key   28
+               :value "28"}
               {:key   32
-               :value "Big"}
+               :value "32"}
               {:key   48
-               :value "Very big"}
+               :value "48"}
               {:key   80
-               :value "Seriously Big!"}]}])
+               :value "80"}]}
+   {:label "Emoji"
+    :key   :emoji
+    :type  :text}
+   {:label   "Customization color:"
```

Comment:
FYI @smohamedjavid, I recently added a function to help us be more DRY. https://github.com/mohsen-ghafouri/status-react/blob/43cba161d66255ec8983f08d74e6050643dd0b34/src/status_im2/contexts/quo_preview/preview.cljs#L226

But I think the better implementation might be to use this `color-picker/color-list` you used in this PR and which I've seen other PRs used as well.

Reply:
Thanks, @ilmotta! This helps a lot.

I'm not sure about using `color-picker/colour-list` as we have `:no-color` option, which we agreed to remove due to some reason/issues from the Desktop app. 

IMO, even the `colour-list` declaration (and usage in preview screens) in the `color-picker` component should get the colours from `quo2.foundations.colors` ns as it is our source of truth.

Reply:
Perhaps we can update the other uses in the preview screens to whatever you decide upon too. 
I agree with your point @smohamedjavid but no strong opinions on this 👍

Reply:
It's not clear to me either, I would just welcome such generalization because I keep seeing PRs repeating this field in preview screens. I'll keep an eye in my next PRs for quo components, and if this is not improved until then I'll try to do something, should be quick.